library/ssl_msg.c

```diff
@@ -86,72 +86,6 @@ int mbedtls_ssl_check_timer( mbedtls_ssl_context *ssl )
     return( 0 );
 }
 
-#if defined(MBEDTLS_SSL_RECORD_CHECKING)
```

Comment:
Actually here we wanted to remove the option but keep the function unconditionally, as you did in #4489 - since that PR was just merged, you'll need to resolve the conflict here by keeping this block.

Reply:
Done